roles/backup/templates/etc/cron.weekly/cnx-slim-dump

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+# Create a slim dump of the cnx production database so developers can use it
+# for development and testing
+
+# 0. Make sure this server can access the production database (with pg_dump or
+#    ssh)
+
+# Target databases settings
+SLIM_DUMP_DB_NAME=cnx_slim_dump
+SLIM_DUMP_FILENAME="/var/backups/$SLIM_DUMP_DB_NAME.$(date +%Y-%m-%d).sql.gz"
+
+# Source databases settings
+ARCHIVE_DB_HOST={{ archive_db_host }}
+ARCHIVE_DB_PORT={{ archive_db_port }}
+ARCHIVE_DB_USER={{ archive_db_user }}
+ARCHIVE_DB_NAME={{ archive_db_name }}
+
+# Only set SSH_USER if pg_dump cannot access the production database
+SSH_USER=
+
+# Install clean up code
+function cleanup {
+    # Only keep 3 copies of slim dump
+    bname=$(basename $SLIM_DUMP_FILENAME)
+    ls -1 -r $(dirname $SLIM_DUMP_FILENAME)/${bname/.*/}* | sed -n '4,$ p' | xargs rm -f
+    psql -U postgres -c "DROP DATABASE IF EXISTS $SLIM_DUMP_DB_NAME;"
+}
+trap cleanup EXIT
+
+# 1. Create a temporary database for the slim dump
+psql -U postgres <<EOF
+DROP DATABASE IF EXISTS $SLIM_DUMP_DB_NAME;
+CREATE DATABASE $SLIM_DUMP_DB_NAME;
```

Comment:
Pretty sure this will fire
```
ERROR:  cannot execute CREATE DATABASE in a read-only transaction
```
 if pointed at a replicant, so should _not_ be set for production.

Reply:
This is using the local database on the backup server.

Reply:
Which is obvious if I read closely - there's no hostname provided, so has to be happening locally.